AnalyticsTests/AnalyticsTests.swift

```diff
@@ -259,4 +259,21 @@ class AnalyticsTests: XCTestCase {
         integration?.test_fileStorage()?.resetAll()
         XCTAssert(integration?.test_queue()?.isEmpty ?? false)
     }
+    
+    func testDeviceTokenRegistration() {
+        func getStringFrom(token: Data) -> String {
+            return token.reduce("") { $0 + String(format: "%02.2hhx", $1) }
```

Comment:
Might be nice to have a comment on this line to explain the format that is being converted